### PR TITLE
Tips, Correct documentation, elaborate, and Apache

### PR DESCRIPTION
**Pull request by adamoutler**
## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_docs/configuration/remote.markdown [hunk 1]
```diff
@@ -15,13 +15,25 @@ Remember to follow the [securing checklist](/docs/configuration/securing/) befor
...
 </div>
 
+## Setting up remote access
 <div class='note'>
-Home Assistant no longer support remote access via IP address since release 0.77, you have to use a domain name.
+Home Assistant no longer supports remote access via IP address since release 0.77, you have to use a domain name.  A Domain can be obtained cheaply from several providers. 
 </div>
 
-The most common approach is to set up port forwarding (for any port) from your router to port 8123 on the computer that is hosting Home Assistant. General instructions on how to do this can be found by searching `<router model> port forwarding instructions`. You can use any free port on your router and forward that to port 8123.
+To set up a remote instance, you will need to set up Network Address Translation, Port Forwarding on your router, to port 8123 on the computer that is hosting Home Assistant.
```
Comment (frenck) [inline source/_docs/configuration/remote.markdown]: I get that these is the correct technical terms, however, a regular end-user has no clue what network address translation is.

Let's just keep it at port forwarding, there is no need to overcomplicate things for those people.

File: source/_docs/configuration/remote.markdown [hunk 2]
```diff
@@ -15,13 +15,25 @@ Remember to follow the [securing checklist](/docs/configuration/securing/) befor
...
 <div class='note'>
-Home Assistant no longer support remote access via IP address since release 0.77, you have to use a domain name.
+Home Assistant no longer supports remote access via IP address since release 0.77, you have to use a domain name.  A Domain can be obtained cheaply from several providers. 
 </div>
 
-The most common approach is to set up port forwarding (for any port) from your router to port 8123 on the computer that is hosting Home Assistant. General instructions on how to do this can be found by searching `<router model> port forwarding instructions`. You can use any free port on your router and forward that to port 8123.
+To set up a remote instance, you will need to set up Network Address Translation, Port Forwarding on your router, to port 8123 on the computer that is hosting Home Assistant. 
 
-A problem with making a port accessible is that some Internet Service Providers only offer dynamic IPs. This can cause you to lose access to Home Assistant while away. You can solve this by using a free Dynamic DNS service like [DuckDNS](https://www.duckdns.org/).
+:::tip
```
Comment (frenck) [inline source/_docs/configuration/remote.markdown]: This syntax doesn't work in Jekyll.

File: source/_docs/configuration/remote.markdown [hunk 3]
```diff
@@ -15,13 +15,25 @@ Remember to follow the [securing checklist](/docs/configuration/securing/) befor
...
 
-The most common approach is to set up port forwarding (for any port) from your router to port 8123 on the computer that is hosting Home Assistant. General instructions on how to do this can be found by searching `<router model> port forwarding instructions`. You can use any free port on your router and forward that to port 8123.
+To set up a remote instance, you will need to set up Network Address Translation, Port Forwarding on your router, to port 8123 on the computer that is hosting Home Assistant. 
 
-A problem with making a port accessible is that some Internet Service Providers only offer dynamic IPs. This can cause you to lose access to Home Assistant while away. You can solve this by using a free Dynamic DNS service like [DuckDNS](https://www.duckdns.org/).
+:::tip 
+General Instructions on Port Forwarding can be found by searching `<router model> port forwarding instructions`. 
+:::
+
+:::warning
```
Comment (frenck) [inline source/_docs/configuration/remote.markdown]: This syntax doesn't work in Jekyll.

File: source/_docs/configuration/remote.markdown [hunk 4]
```diff
@@ -15,13 +15,25 @@ Remember to follow the [securing checklist](/docs/configuration/securing/) befor
...
+It is not advised to set up direct NAT without HTTPS enabled.  Always enable HTTPS.
+:::
+
+One potential problem with port forwarding on a residential connection is most Internet Service Providers utilize dynamic IP addresses.  This offers the security of making it harder to find your IP address in the continuously moving sea of other IP addresses. However, it also makes it more difficult to find your Home Assistant within that sea. . You can solve this by using a free Dynamic DNS service like 
+:::tip
```
Comment (frenck) [inline source/_docs/configuration/remote.markdown]: This syntax doesn't work in Jekyll.

File: source/_docs/configuration/remote.markdown [hunk 5]
```diff
@@ -15,13 +15,25 @@ Remember to follow the [securing checklist](/docs/configuration/securing/) befor
...
-A problem with making a port accessible is that some Internet Service Providers only offer dynamic IPs. This can cause you to lose access to Home Assistant while away. You can solve this by using a free Dynamic DNS service like [DuckDNS](https://www.duckdns.org/).
+:::tip 
+General Instructions on Port Forwarding can be found by searching `<router model> port forwarding instructions`. 
+:::
+
+:::warning
+It is not advised to set up direct NAT without HTTPS enabled.  Always enable HTTPS.
+:::
+
+One potential problem with port forwarding on a residential connection is most Internet Service Providers utilize dynamic IP addresses.  This offers the security of making it harder to find your IP address in the continuously moving sea of other IP addresses. However, it also makes it more difficult to find your Home Assistant within that sea. . You can solve this by using a free Dynamic DNS service like
```
Comment (frenck) [inline source/_docs/configuration/remote.markdown]:
```suggestion
One potential problem with port forwarding on a residential connection is most internet service providers use dynamic IP addresses. This offers the security of making it harder to find your IP address in the continuously moving sea of other IP addresses. However, it also makes it more difficult to find your Home Assistant within that sea. You can solve this by using a free Dynamic DNS service like
```

> This offers the security of making it harder to find your IP address in the continuously moving sea of other IP addresses.

That is not the sole reason behind this. Also, I think the text stays the same when removing that line. So let's do that.

File: source/_docs/configuration/remote.markdown [hunk 6]
```diff
@@ -30,3 +42,71 @@ If you cannot access your Home Assistant installation remotely, remember to chec
 Just putting a port up is not secure. You should definitely consider encrypting your traffic if you are accessing your Home Assistant installation remotely. For details please check the [set up encryption using Let's Encrypt](/blog/2017/09/27/effortless-encryption-with-lets-encrypt-and-duckdns/) blog post or this [detailed guide](/docs/ecosystem/certificates/lets_encrypt/) to using Let's Encrypt with Home Assistant.
 
 </div>
+
+
+## Apache Proxy
```
Comment (frenck) [inline source/_docs/configuration/remote.markdown:46]: All reverse proxy documentation has been removed from our end-user documentation. You can find, contribute or create them on the Community Guides section of the Community forums.

Please remove this section.

File: source/_docs/configuration/remote.markdown [hunk 7]
```diff
@@ -15,13 +15,25 @@ Remember to follow the [securing checklist](/docs/configuration/securing/) befor
 
 </div>
 
+## Setting up remote access
 <div class='note'>
-Home Assistant no longer support remote access via IP address since release 0.77, you have to use a domain name.
+Home Assistant no longer supports remote access via IP address since release 0.77, you have to use a domain name.  A Domain can be obtained cheaply from several providers.
```
Comment (frenck) [inline source/_docs/configuration/remote.markdown]:
```suggestion
Home Assistant no longer supports remote access via IP address since release 0.77, you have to use a domain name. A Domain can be obtained from several providers. 
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_docs/configuration/remote.markdown (hunk 2 above)
Comment (adamoutler) [reply source/_docs/configuration/remote.markdown]: This is supported elsewhere on the site does it require a div here?

Comment (adamoutler) [reply source/_docs/configuration/remote.markdown]: I see that a preview is generated.  Where can I find this preview?

Comment (frenck) [reply source/_docs/configuration/remote.markdown]: https://developers.home-assistant.io/docs/documenting/create-page#html

File: source/_docs/configuration/remote.markdown (hunk 1 above)
Comment (adamoutler) [reply source/_docs/configuration/remote.markdown]: removed "Network Address Translation"

File: source/_docs/configuration/remote.markdown [hunk 8]
```diff
@@ -25,13 +25,13 @@ To set up a remote instance, you will need to set up Port Forwarding on your rou
 <div class='note'>
 General Instructions on Port Forwarding can be found by searching `<router model> port forwarding instructions`. 
 </div>
-<div class='note important'>
+<div class='note warning'>
 It is not advised to set up direct NAT without HTTPS enabled.  Always enable HTTPS.
 </div>
```
Comment (frenck) [inline source/_docs/configuration/remote.markdown]: Do we really 2 big blocks so close behind each other?

File: source/_docs/configuration/remote.markdown [hunk 9]
```diff
@@ -25,13 +25,13 @@ To set up a remote instance, you will need to set up Port Forwarding on your rou
...
 
 One potential problem with port forwarding on a residential connection is most internet service providers use dynamic IP addresses. This offers the security of making it harder to find your IP address in the continuously moving sea of other IP addresses. However, it also makes it more difficult to find your Home Assistant within that sea. You can solve this by using a Dynamic DNS service.
 <div class='note'>
-[DuckDNS](https://www.duckdns.org/) provides free Dynamic DNS. [Google Domains](https://domains.google), or [No IP](https://noip.com) provide both the Domain and the Dynamic DNS required for Home Assistant. 
+ [DuckDNS](https://www.duckdns.org/) provides free Dynamic DNS. [Google Domains](https://domains.google), or [No IP](https://noip.com) provide both the Domain and the Dynamic DNS required for Home Assistant. 
 </div>
```
Comment (frenck) [inline source/_docs/configuration/remote.markdown]: I don't think this should be in a note block. This can be done in regular text, as there is not need to alarm the user for these links.

File: source/_docs/configuration/remote.markdown (hunk 6 above)
Comment (adamoutler) [reply source/_docs/configuration/remote.markdown:46]: Nabu Casa business decision?

File: source/_docs/configuration/remote.markdown [hunk 10]
```diff
@@ -31,7 +31,7 @@ It is not advised to set up direct NAT without HTTPS enabled.  Always enable HTT
 
 One potential problem with port forwarding on a residential connection is most internet service providers use dynamic IP addresses. This offers the security of making it harder to find your IP address in the continuously moving sea of other IP addresses. However, it also makes it more difficult to find your Home Assistant within that sea. You can solve this by using a Dynamic DNS service.
 <div class='note'>
- [DuckDNS](https://www.duckdns.org/) provides free Dynamic DNS. [Google Domains](https://domains.google), or [No IP](https://noip.com) provide both the Domain and the Dynamic DNS required for Home Assistant. 
+ <a href="https://www.duckdns.org/">DuckDNS</a> provides free Dynamic DNS. <a href="https://domains.google">Google Domains</a>, or <a href="https://noip.com">No IP</a> provide both the Domain and the Dynamic DNS required for Home Assistant.
```
Comment (frenck) [inline source/_docs/configuration/remote.markdown:34]: Don't use HTML, use markdown. Please revert this change.

File: source/_docs/configuration/remote.markdown (hunk 9 above)
Comment (adamoutler) [reply source/_docs/configuration/remote.markdown]: I thought class=warning was for alarm?  This is just a note.

File: source/_docs/configuration/remote.markdown (hunk 6 above)
Comment (frenck) [reply source/_docs/configuration/remote.markdown:46]: No, a Home Assistant project decision. As we became documentation for about every reverse proxy out there. A reverse proxy is optional, and not a core task of the project. Hence, moved out of the user documentation.

File: source/_docs/configuration/remote.markdown (hunk 9 above)
Comment (frenck) [reply source/_docs/configuration/remote.markdown]: Sure, but avoiding them is in general a good idea. There is no need to emphasize these on this page, as part of a regular text it would still work as well.

**Comment by frenck** [comment]
@adamoutler? Why did you close this PR?

**Comment by adamoutler** [comment]
> @adamoutler? Why did you close this PR?

Not feeling it anymore.  As a user, the only thing I would want on this page is removed by Home Assistant decision.  In its decided state, this documentation is not useful to me and I don't care to update it.  

My goal of any documentation contribution is to make it useful to myself in the future.  If I need to keep notes or reference the information from another source, then I feel it belongs in documentation.   

If the decision is to remove the Apache example, then I do not feel the need to contribute to this particular page.

**Comment by frenck** [comment]
Alright, sorry to hear you feel that way. I think your other additions were still valuable.

**Comment by adamoutler** [comment]
Let's make this easier for me then.  Make all the changes you want.  I don't want to put work into it to figure things out.

File: source/_docs/configuration/remote.markdown (hunk 6 above)
Comment (adamoutler) [reply source/_docs/configuration/remote.markdown:46]: There is a warning which states the current documentation is not recommended for use.  For this reason, this documentation is not useful.

**Comment by frenck** [comment]
> I don't want to put work into it to figure things out.

Sorry, that I won't do. Let's close it.
